ee-9/feature-pack/pom.xml

```diff
@@ -0,0 +1,1182 @@
+  ~  * See the License for the specific language governing permissions and
+  ~  * limitations under the License.
+  ~  */
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.wildfly</groupId>
+        <artifactId>wildfly-ee-9-parent</artifactId>
+        <!--
+        Maintain separation between the artifact id and the version to help prevent
+        merge conflicts between commits changing the GA and those changing the V.
+        -->
+        <version>22.0.0.Beta1-SNAPSHOT</version>
+    </parent>
+    
+    <artifactId>wildfly-preview-feature-pack</artifactId>
+
+    <name>WildFly: EE 9 Preview Galleon Feature Pack</name>
+    <description>Tech Preview Galleon feature pack providing full platform Jakarta EE 9 functionality</description>
+    <packaging>pom</packaging>
+
+    <properties>
+        <license.directory>${project.build.directory}/resources/content/docs/licenses</license.directory>
+        <mp.galleon.common.resources.directory>${basedir}/../../microprofile/galleon-common/src/main/resources</mp.galleon.common.resources.directory>
+        <mp.license.directory>${basedir}/../../microprofile/galleon-common/src/license</mp.license.directory>
+        <full.common.resources.directory>${basedir}/../../ee-feature-pack/common/src/main/resources</full.common.resources.directory>
+        <full.galleon-common.resources.directory>${basedir}/../../ee-feature-pack/galleon-common/src/main/resources</full.galleon-common.resources.directory>
+        <full.common.license.directory>${basedir}/../../ee-feature-pack/common/src/license</full.common.license.directory>
+        <servlet.common.resources.directory>${basedir}/../../servlet-feature-pack/common/src/main/resources</servlet.common.resources.directory>
+        <servlet.galleon-common.resources.directory>${basedir}/../../servlet-feature-pack/galleon-common/src/main/resources</servlet.galleon-common.resources.directory>
+        <servlet.common.license.directory>${basedir}/../../servlet-feature-pack/common/src/license</servlet.common.license.directory>
+        <full.ee-9-api.license.directory>${basedir}/src/license</full.ee-9-api.license.directory>
+
+
+        <version.com.sun.faces>3.0.0.SP01</version.com.sun.faces>
+        <version.com.sun.activation.jakarta.activation>2.0.0-RC3</version.com.sun.activation.jakarta.activation>
+        <version.jakarta.annotation.jakarta-annotation-api>2.0.0-RC1</version.jakarta.annotation.jakarta-annotation-api>
+        <version.jakarta.authorization.jakarta-authorization-api>2.0.0-RC1</version.jakarta.authorization.jakarta-authorization-api>
+        <version.jakarta.batch.jakarta.batch-api>2.0.0-M6</version.jakarta.batch.jakarta.batch-api>
```

Comment:
I noticed jakarta.batch:jakarta.batch-api:jar:2.0.0 is listed on https://jakarta.ee/specifications/batch/2.0, I switched locally to <version.jakarta.batch.jakarta.batch-api>2.0.0</version.jakarta.batch.jakarta.batch-api> and could build.  So we probably can update the others as well (we just have to look up the exact version).

Reply:
@scottmarlow I did an update last night of everything we override in this pom where there was a new version in maven central.